Dockerfile

```diff
@@ -51,27 +51,27 @@ RUN --mount=target=. \
     -o /go/bin/goshimmer
 
 # Docker cache will be invalidated for RUNs after ARG definition (https://docs.docker.com/engine/reference/builder/#impact-on-build-caching)
-ARG CUSTOM_SNAPSHOT
+ARG CUSTOM_SNAPSHOT_URL
+ARG FEATURE_DEFAULT_SNAPSHOT=0
 
 # Enable building the image without downloading the snapshot.
 # It's possible to download custom snapshot from external storage service - necessary for feature network deployment.
 # If built with dummy snapshot then a snapshot needs to be mounted into the resulting image.
-RUN if [ "$DOWNLOAD_SNAPSHOT" -gt 0 ] && [ "$CUSTOM_SNAPSHOT" = "" ]; then \
-    wget -O /tmp/snapshot.bin https://dbfiles-goshimmer.s3.eu-central-1.amazonaws.com/snapshots/nectar/snapshot-latest.bin ;  \
-    echo "default" > /tmp/test.txt;\
-    elif [ "$DOWNLOAD_SNAPSHOT" -gt 0 ] && [ "$CUSTOM_SNAPSHOT" != "" ]; then \
-    apt update; apt install -y gawk; \
-    git clone https://github.com/stck-lzm/badown.git; \
-    cd badown; \
-    ./badown "$CUSTOM_SNAPSHOT"; \
-    SNAPSHOT_FILE=$(ls -t *.bin | head -1); \
-    mv "$SNAPSHOT_FILE" /tmp/snapshot.bin; \
-    echo "custom" > /tmp/test.txt;\
+RUN if [ "$DOWNLOAD_SNAPSHOT" -gt 0 ] && [ "$CUSTOM_SNAPSHOT_URL" = "" ] && [ "$FEATURE_DEFAULT_SNAPSHOT" -eq 0 ]; then \
+      wget -O /tmp/snapshot.bin https://dbfiles-goshimmer.s3.eu-central-1.amazonaws.com/snapshots/nectar/snapshot-latest.bin ;  \
+    elif [ "$DOWNLOAD_SNAPSHOT" -gt 0 ] && [ "$CUSTOM_SNAPSHOT_URL" = "" ] && [ "$FEATURE_DEFAULT_SNAPSHOT" -gt 0 ]; then \
+      wget -O /tmp/snapshot.bin https://dbfiles-goshimmer.s3.eu-central-1.amazonaws.com/snapshots/feature/snapshot.bin;  \
+    elif [ "$DOWNLOAD_SNAPSHOT" -gt 0 ] && [ "$CUSTOM_SNAPSHOT_URL" != "" ]; then \
+      apt update; apt install -y gawk; \
+      git clone https://github.com/ffluegel/zippyshare.git; \
```

Comment:
Why did you eventually change the script to only support zippyshare?
If zippyshare is the most convenient and easy-to-use service I have nothing against it =)

Reply:
The previous script did not download the actual file from Zippyshare but some HTML contents which I realized only after I tried deploying the network and all nodes failed. 

I think zippyshare is quite convenient as you don't need to create an account or anything, just upload the file with 500MB limit, which is plenty for our use, and you get a download link that you can use in the Action. Can't get any simpler than that :smile:

Reply:
:+1: